realm/realm-library/src/main/java/io/realm/Realm.java

```diff
@@ -250,73 +236,37 @@ private static synchronized Realm create(RealmConfiguration configuration) {
                 migrateRealm(configuration);
             }
 
-            return createAndValidate(configuration, true, autoRefresh);
+            return createAndValidate(configuration, columnIndices);
```

Comment:
Is there any case that `columnIndices` is valid after migrating?

Is `columnIndices` always `null` if reach here?